sample-apps/react/react-dogfood/hooks/useNotificationSounds.ts

```diff
@@ -0,0 +1,35 @@
+import { useCall } from '@stream-io/video-react-sdk';
+import { useCallback, useEffect } from 'react';
+import { beep } from '../lib/beeper';
+
+export function useNotificationSounds() {
+  const call = useCall();
+  // We don't want to play the sound when the user joins themself
+  const isSelf = useCallback(
+    (userId: string) => userId !== call?.streamClient.userID,
+    [call],
+  );
+
+  useEffect(() => {
+    if (!call) {
+      return;
+    }
+
+    const unlistenJoin = call.on('call.session_participant_joined', (event) => {
+      if (!isSelf(event.participant.user.id)) {
```

Comment:
I think you can also use: `call.currentUserId` here. It should simplify the implementation.

Reply:
Oh, didn't know it exists. Cool!